strawberry_django_plus/type.py

```diff
@@ -89,13 +91,18 @@ def _from_django_type(
 
         field = cast(StrawberryDjangoField, field)
 
-        # FIXME: Improve this...
         if not field.base_resolver:
 
-            def conn_resolver(root):
-                return getattr(root, name).all()
+            def conn_resolver(type_annotation, root, info: Info):
+                qs =  getattr(root, name).all()
+                remote_type_defs = get_args(type_annotation.annotation)
+                remote_type = eval_type(remote_type_defs[0], type_annotation.namespace, None)
+                if hasattr(remote_type, "get_queryset"):
+                    return remote_type.get_queryset(qs, info)
+                return qs
+            aware_conn_resolver = partialfunc(conn_resolver, type_annotation)
```

Comment:
`type_annotation` may be `None` here, but `conn_resolver` expects it not to be

Reply:
Handled